docs/data-sources/background_worker.md

```diff
@@ -109,6 +110,14 @@ Read-Only:
 - `preview_notifications_enabled` (String) Whether notifications for previews of this service are sent.
 
 
+<a id="nestedatt--previews"></a>
+### Nested Schema for `previews`
```

Comment:
Will we need to do this for our other added fields as well? 
I'm assuming no(?) since the pre-migration versions didnt exist here either it looks like.

Reply:
PreviewsExpireAfterDays --> previews.ExpireAfterDays. 
I assume we don't have to do anything about it here cuz PreviewsExpireAfterDays didn't exist in the previous world anyways

Reply:
Yes, this is not exposed as part of the REST API.